test/lib/mocksdp.js

```diff
@@ -74,6 +74,7 @@ a=msid:- ${id}\r
 `) + `\
 a=ssrc:${ssrc} cname:0\r
 a=ssrc:${ssrc} msid:${type === 'planb' ? 'stream' : '-'} ${id}\r
+a=mid:mid_${id}\r
```

Comment:
@syerrapragada This will result in two `a=mid` lines in case of unified plan. Can you make sure that doesn't happen?